container/Dockerfile

```diff
@@ -38,19 +38,19 @@ ENTRYPOINT [ "/opt/dirac/dockerEntrypoint.sh" ]
 # Create self-signed host certificate from auto-generated CA
 WORKDIR /opt/dirac/etc/grid-security
 RUN source /opt/dirac/bashrc && openssl genrsa -out hostkey.pem 2048
-RUN curl -L -o openssl_config https://raw.githubusercontent.com/fstagni/DIRAC/devDocsDocker-1/tests/openssl_config_DIRACDockerDevBox && \
+RUN curl -L -o openssl_config https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/tests/openssl_config_DIRACDockerDevBox && \
```

Comment:
Rather than `integration`, wouldn't it be better to take the one in the same version as we installed ?